packages/components/inputs/money-input/README.md

```diff
@@ -49,29 +49,58 @@ export default Example;
+| `isAutofocussed`        | `boolean`                                                                                             |          |           | Focus the input on initial render                                                                                                                             |
+| `onChange`              | `Function`<br/>[See signature.](#signature-onChange)                                                  |    ✅    |           | Called with the event of the input or dropdown when either the currency or the amount have changed.&#xA;<br />&#xA;Signature: `(event) => void`               |
+| `menuPortalTarget`      | `ReactSelectProps['menuPortalTarget']`                                                                |          |           | Dom element to portal the currency select menu to&#xA;[Props from React select was used](https://react-select.com/props)                                      |
+| `menuPortalZIndex`      | `number`                                                                                              |          | `1`       | z-index value for the currency select menu portal                                                                                                             |
+| `menuShouldBlockScroll` | `ReactSelectProps['menuShouldBlockScroll']`                                                           |    ✅    |           | whether the menu should block scroll while open&#xA;[Props from React select was used](https://react-select.com/props)                                        |
+| `hasError`              | `boolean`                                                                                             |          |           | Indicates that input has errors                                                                                                                               |
+| `hasWarning`            | `boolean`                                                                                             |          |           | Control to indicate on the input if there are selected values that are potentially invalid                                                                    |
+| `hasHighPrecisionBadge` | `boolean`                                                                                             |          |           | Shows high precision badge in case current value uses high precision.                                                                                         |
+| `horizontalConstraint`  | `union`<br/>Possible values:<br/>`, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 'scale', 'auto'` |          | `'scale'` | Horizontal size limit of the input fields.                                                                                                                    |
+
+## Signatures
+
+### Signature `value`
+
+```ts
+{
+  amount: string;
+  currencyCode: TCurrencyCode;
+}
+```
+
+### Signature `currencies`
```

Comment:
This signature section is blank.
The property is already explained in the table above so maybe we can remove this sub-heading.
What do you think?

Reply:
I just realised that types that are an array of values don't appear in the signature section. 
I'm not sure why. 
@emmenko, do you have ideas why that is happening?

Reply:
Did you re-run the readme generator?

Reply:
Yes, I did. 
This is also an issue in some other parts of the app.
Take a look at [data-table](https://github.com/commercetools/ui-kit/tree/main/packages/components/data-table) also

Reply:
Hmm I see...well actually `Row` is a generic type, that's why the signature is empty.

Nevertheless, we might want to adjust the readme generator to trying showing more useful information. We can also avoid linking empty signatures. The field description should already provide the basic information.

As for the `currencies` field, it's a `string[]` so I don't know why it thinks there is a need for a signature. We would need to debug that.

Reply:
Sure. I will create an issue for that.